mesos-tools/src/cgcloud/mesos_tools/__init__.py

```diff
@@ -374,3 +375,10 @@ def __prepare_slave_args( self ):
             if attributes:
                 attributes = ';'.join( '%s:%r' % i for i in attributes.items( ) )
                 f.write( "--attributes='%s'" % attributes )
+
+    def __prepare_credentials(self):
+        # Create the credentials file and transfer ownership to mesosbox
+        with open('/var/lib/mesos/credentials.txt', 'w') as f:
+            f.write('toil liot')
+        check_call(['chown', 'mesosbox:mesosbox', '/var/lib/mesos/credentials.txt'])
```

Comment:
I'll do it and then should i remove it from toil-jenkins-slave (since it inherits mesosbox). This might be related to #114